config/initializers/time_formats.rb

```diff
@@ -0,0 +1 @@
+Time::DATE_FORMATS[:long_ordinal_12h] = lambda { |time| time.strftime("%B #{time.day.ordinalize}, %Y %l:%M %p") }
```

Comment:
Nice!  I've never seen this one before.  That's really cool that it can take a lambda.